.github/workflows/test.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       id-token: "write"
     services:
       postgres:
-        image: postgres:15.4
+        image: pgvector/pgvector:pg15
```

Comment:
This fixes Postgres to version 15 but it does not fix the version on the pgvector extension. There are tags to fix both, the version of the extension + the version of the database. See [here](https://hub.docker.com/r/pgvector/pgvector/tags?page=1&name=pg15). I strongly recommend we use tags like [0.6.2-pg15](https://hub.docker.com/layers/pgvector/pgvector/0.6.2-pg15/images/sha256-0d3d57e65ca0d75f7d2c5f492f5c06353543495b2766d3998344708c2e8c8a04?context=explore).

CloudSQL uses, for now, the version 0.5.1 as you can see [here](https://cloud.google.com/sql/docs/postgres/release-notes), which isn't available as Docker tag. 

I propose we use 0.6.2-pg15 as I expect Google to upgrade to it sooner than later and it is a fixed version.

Reply:
nice! i'll fix this in build.yml too